package.json

```diff
@@ -6,30 +6,41 @@
   "repository": "alex-e-leon/print-tree",
   "author": "alex leon <alex@leon.st>",
   "license": "MIT",
+  "files": [
+    "lib"
```

Comment:
whitelist the `lib` directory as that's the only thing we need to publish

Reply:
👍  I think this is actually so much better than .npmignore